docs/sphinx/conf.py.in

```diff
@@ -128,15 +113,14 @@ def popen(args, stdin=None, stdout=subprocess.PIPE, stderr=subprocess.PIPE):
 #modindex_common_prefix = []
 
 # Variables used to define Github extlinks
-if "SOURCE_BRANCH" in os.environ and len(os.environ.get('SOURCE_BRANCH')) > 0:
-    source_branch = os.environ.get('SOURCE_BRANCH')
-else:
-    source_branch = 'develop'
-
-if "SOURCE_USER" in os.environ and len(os.environ.get('SOURCE_USER')) > 0:
-    user = os.environ.get('SOURCE_USER')
-else:
-    user = 'openmicroscopy'
+source_branch = 'develop'
+if (ext_source_branch is not None and len(ext_source_branch) > 0 and
+    ext_source_branch != '${sphinx_source_branch}'):
```

Comment:
Why this extra check? If `ext_source_branch` is passed via `-Dsphinx,source.branch` we likely want it to be able to override the default value.